src/main/java/bio/terra/common/sam/exception/SamExceptionFactory.java

```diff
@@ -59,4 +59,9 @@ public static ErrorReportException create(String messagePrefix, ApiException api
         return new SamInternalServerErrorException(message, apiException);
     }
   }
+
+  public static ErrorReportException create(
+      String messagePrefix, InterruptedException interruptedException) {
```

Comment:
nit: worth duplicating some of the comment on SamInterruptedException on when this method should be thrown, or just don't have it in this factory since there's nothing wrong with the constructor and we're not trying to hide something about creation.

Reply:
Added comment.